components/shared_code/src/shared_data_model/metrics.py

```diff
@@ -44,6 +44,23 @@
         sources=["azure_devops", "jira"],
         tags=[Tag.PROCESS_EFFICIENCY],
     ),
+    "change_failure_rate": Metric(
+        name="Change failure rate",
+        description="The percentage of deployments causing a failure in production.",
+        rationale="The change failure rate is an indicator of the DevOps effectiveness of a team.",
+        documentation="""Because this metric is a "rate", it needs a numerator (the number of failed deployments
+in the look back period) and a denominator (the number of deployments in the look back period). The metric can be
+configured with a combination of Jira, and either Jenkins or GitLab CI as source.
+
+Configuring Jira as source for the metric, means that it is used as the number of failures (numerator). It needs to be
+configured with either Jenkins or GitLab CI as source for the number of deployments (denominator).""",
```

Comment:
```suggestion
configured with a combination of Jira, and either Jenkins or GitLab CI as source. Jira is used as source for the number of failures (numerator). Either Jenkins or GitLab CI is used as source for the number of deployments (denominator).""",
```

Reply:
This does read more clearly, but the originally intended help text in #6722 was:
```The metric can either be configured with Azure DevOps as a source, or a combination of Jira and Jenkins or GitLab CI as source. Configuring Azure DevOps as source for the metric, means that it is used both for number of failures (numerator) and number of deployments (denominator). Azure DevOps needs to be added as source only once. Configuring Jira as source for the metric, means that it is used as the number of failures (numerator). It needs to be configured with either Jenkins or GitLab CI as source for the number of deployments (denominator).```